src/qibo/backends/profiles.yml

```diff
@@ -13,9 +13,9 @@ backends:
     from: qibo.backends.tensorflow
 
   # hardware backends
-  - name: qiboicarusq
-    class: IcarusQBackend
-    from: qibo.backends.hardware
+  - name: qibolab
+    class: QibolabBackend
+    from: qibolab.backend
```

Comment:
Just a question about this point that I forgot to discuss. What do you think if we change this with:
```yaml
- name: qibolab
  driver: qibolab.backend.QibolabBackend
```

Reply:
I am fine with that, particularly given that `from` and `class` are Python words so it may be preferrable to avoid them. If you like I can open a new PR implementing this. If I understand correclty you mean substituting both the `from` and `class` entries with one `driver` entry, right?

Reply:
Indeed, sounds reasonable, and makes sense to update this.